### PR TITLE
Bug 1881016 - Added experimental onboarding strings for Nimbus treatments C & D

### PR DESCRIPTION
**Pull request by t-p-white**
⚠️ https://github.com/mozilla-mobile/firefox-android/pull/5666 should be reviewed before this as the string numbering is continued from there 

ℹ️  Changes from experiment Doc:
- **Branch D**
  - Copy changed from “600” to “700” for Fenix based on the real number of extensions available: 782
  - “PDF’s” was changed to “PDFs”
- **Branch C**   
  - The period after “Firefox privacy notice” has been omitted

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1881016

### CODE REVIEW

File: fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -317,9 +317,13 @@
     <!-- Text for the button to set firefox as default browser on the device -->
     <!-- Title for set firefox as default browser screen used by Nimbus experiments. -->
     <string name="juno_onboarding_default_browser_title_nimbus_2">We love keeping you safe</string>
+    <!-- Title for set firefox as default browser screen used by Nimbus experiments. -->
+    <string name="juno_onboarding_default_browser_title_nimbus_4" tools:ignore="UnusedResources">Safe browsing with more choices</string>
     <!-- Description for set firefox as default browser screen used by Nimbus experiments. -->
     <string name="juno_onboarding_default_browser_description_nimbus_3">Our non-profit backed browser helps stop companies from secretly following you around the web.</string>
     <!-- Description for set firefox as default browser screen used by Nimbus experiments. -->
+    <string name="juno_onboarding_default_browser_description_nimbus_5" tools:ignore="UnusedResources">Known trackers? Blocked automatically. Extensions? Try all 600. PDF’s? Our built-in reader makes them easy to manage.</string>
```
Comment (Delphine) [inline fenix/app/src/main/res/values/strings.xml:325]: There are currently 766 add-ons available: https://addons.mozilla.org/en-US/android/search/
Yet this string calls out 600.
We might want to double check if 600 is really the number we want to call out here

Comment (t-p-white) [reply fenix/app/src/main/res/values/strings.xml]: @Delphine thanks for asking this 🙂 per the latest comms this should be 700

Comment (ewachowiak) [reply fenix/app/src/main/res/values/strings.xml]: It looks like there's a typo as well. PDF's should be PDFs. Thanks!

Comment (t-p-white) [reply fenix/app/src/main/res/values/strings.xml]: Thanks @ewachowiak have updated that copy

**Review by Amejia481 — APPROVED** [review]
LGTM!

**Review by Delphine — APPROVED** [review]

**Comment by t-p-white** [comment]
@Mergifyio backport releases_v124

**Comment by dsmithpadilla** [comment]
@mergify backport releases_v124

**Pull request merged**